core/domain/exp_domain.py

```diff
@@ -1776,6 +1776,67 @@ def create_default_state(
                 default_dest_state_name),
             feconf.DEFAULT_CONTENT_IDS_TO_AUDIO_TRANSLATIONS)
 
+    @classmethod
+    def convert_state(cls, state_dict, conversion_fn):
```

Comment:
Rename to "convert_html_fields_in_state". That's the crux of what this method is doing, yet "html fields" is not mentioned anywhere in the method name.

Reply:
Done!